src/Security/Security.php

```diff
@@ -599,87 +747,172 @@ public function searchUsers(array $filters, array $options = [])
      * @param string $id Unique profile identifier
      * @param array $policies List of policies to apply to this profile
      * @param array $options Optional arguments
+     *
```

Comment:
To be consistent with other SDKs, the second argument  should be an associative array `$body` , which contains a `policies` key.